src/Microsoft.Diagnostics.Runtime/src/DataTargets/ClrInfoProvider.cs

```diff
@@ -45,7 +45,7 @@ public static bool IsSupportedRuntime(ModuleInfo moduleInfo, out ClrFlavor flavo
             if (moduleName is null)
                 return false;
 
-            switch (moduleName)
+            switch (moduleName.ToUpperInvariant())
```

Comment:
This is an ok fix.  Instead of switching on module name should we be converting this to if statements using `moduleName.Equals(..., OrdinalIngoreCase)`?  Or will that not matter here?

Reply:
Yes, I like that better.